requirements.txt

```diff
@@ -1,6 +1,16 @@
 certifi==2021.10.8
+cfgv==3.3.1
 charset-normalizer==2.0.9
+distlib==0.3.4
+filelock==3.4.2
+identify==2.4.1
 idna==3.3
+nodeenv==1.6.0
+platformdirs==2.4.1
+pre-commit==2.16.0
```

Comment:
Since our use case is relatively simple, I'd recommend only pinning the requirements we actually use and letting pip figure out the dependencies, e.g.:

```
pre-commit==2.16.0
PyYAML==6.0
requests==2.26.0
```